src/routes/gifs/unlike-gif.e2e.test.js

```diff
@@ -0,0 +1,118 @@
+const request = require('supertest')
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+const app = require('../../app')
+
+describe('UNLIKE /gif', () => {
```

Comment:
call this something related to the endpoint
```suggestion
describe('DELETE /gifs/:id/:userId', () => {
```